### DISCUSSION TITLE
Guidance on how to dynamically output onProgress()

### DISCUSSION THREAD

**Discussion by mdleung** [norkunas/youtube-dl-php#218] [General]
Hi there!

I'm wondering if there is a way to dynamically update the speed and ETA in the browser.

With my web host, I only have access to Python and PHP interpreters.

Thanks :)

**Comment by norkunas** [comment]
Hi,
If you can't run tools like [mercure](https://mercure.rocks/) or web sockets, then I guess you could create your own entrypoint where you coud leverage [Event-Stream](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events)
